platform/broadcom/sonic-platform-modules-mitac/ly1200-32x/opt/watchdog/watchdog.conf

```diff
@@ -21,7 +21,7 @@
 #test-binary		=
 #test-timeout		=
 
-watchdog-device	= /dev/watchdog
+watchdog-device	= /dev/watchdog5
```

Comment:
can you explain what does the watchdog provides you?

Reply:
It may work for everyone, but we have no plan to modify any existed docker in SONiC.
For watchdog service, the general watchdog of the Linux operating system can meet our requirement.

Reply:
the right place to add watchdog service is pmon docker, we do not have plan to add it in the base image.